vyper/exceptions.py

```diff
@@ -127,13 +128,18 @@ def format_annotation(self, value):
             module_node = node.module_node
 
             # TODO: handle cases where module is None or vy_ast.Module
-            if module_node.get("path") not in (None, "<unknown>"):
-                node_msg = f'{node_msg}contract "{module_node.path}:{node.lineno}", '
+            if module_node.get("resolved_path") not in (None, "<unknown>"):
+                node_msg = self._format_contract_details(
+                    node_msg, module_node.resolved_path, node.lineno
+                )
 
             fn_node = node.get_ancestor(vy_ast.FunctionDef)
             if fn_node:
                 node_msg = f'{node_msg}function "{fn_node.name}", '
 
+        elif self.path is not None:
+            node_msg = self._format_contract_details(node_msg, self.path, node.lineno)
```

Comment:
i think we can move this above line 131 and then simplify the check on line 131?

Reply:
never mind -- it's fine the way it is